### PR TITLE
SCALRCORE-18737 Workspaces > Control dry-run availability

### PR DESCRIPTION
**Pull request by ablik**
### Changelog
* [ ] I have updated CHANGELOG.md according to [CONTRIBUTING.md](../CONTRIBUTING.md)

### Documentation
* [ ] I have updated resource/data source documentation in [docs](../docs)

### State
* [ ] My changes affect the state
* [ ] I have included a state migration and a unit test for it

### CODE REVIEW

**Review by mermoldy — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- `scalr_workspace`: new attribute `vcs_repo.enable_dry_runs` ([#70](https://github.com/Scalr/terraform-provider-scalr/pull/70))
```
Comment (emocharnik) [inline CHANGELOG.md:12]: @ablik why you didn't name provider attribute the same as API one?

Comment (emocharnik) [reply CHANGELOG.md:12]: Looks like bug in the changelog only

Comment (emocharnik) [reply CHANGELOG.md:12]: Also you didn't specify required server version. See examples below.

Comment (recipe) [reply CHANGELOG.md:12]: оно же должно было быть `dry_runs_enabled`

Comment (ablik) [reply CHANGELOG.md]: @recipe of course, its was typo in changelog, now fixed. In project and documentation used `dry_runs_enabled`
@emocharnik added required server version, but have one question. We have two fixes in this release one of them needs `8.0.1-beta.20210407` and second `8.0.1-beta.20210810`, do we need to specify this requirements for each changes or can leave only one newest version for all changes?

**Pull request merged**